src/main/java/sirius/biz/tycho/academy/AcademyVideo.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.tycho.academy;
+
+import sirius.db.mixing.Mapping;
+
+/**
+ * Represents a video which is part of a video academy managed by the {@link OnboardingEngine}.
+ * <p>
+ * Note that an <tt>AcademyVideo</tt> provides the metadata of a video to show and gets multiplied into many
+ * {@link OnboardingVideo onboarding videos} - one for each participant.
+ */
+public interface AcademyVideo {
+
+    /**
+     * Contains the database independent metadata of the video.
+     */
+    Mapping ACADEMY_VIDEO_DATA = Mapping.named("academyVideoData");
+
```

Comment:
Remove empty line

Reply:
we commonly clamp the mapping and field together. As here, we only have the mapping and the getter I'd rather keep them one line apart...

Reply:
Other classes (even in this PR) have the same structure without the empty line. So which one is preferred?

Reply:
Inside this PR mappings are formatted consistently, with a extra line between mapping and getter, without extra line for fields. Or am i missing something?